packages/shared/src/components/drawers/ContextMenuDrawer.tsx

```diff
@@ -55,7 +53,7 @@ export function ContextMenuDrawer({
                 className={classes}
                 onClick={(event) => {
                   action({ value: label, index, event });
-                  ref.current.onClose();
+                  drawerProps.onClose?.(event);
```

Comment:
We already have `onClose` in the `drawerProps` that we can use to close the context menu. No need for having a `ref` anymore.

Reply:
Can you do some sanity checks on other context menus just to make sure they aren't broken after this?

Reply:
I did sanity check post context menu and squad context menu, seems to be working as expected.

Reply:
Nice! Thank you! 🚀

Reply:
I think the only thing we will lose here is the animation if we are not using the `ref`.

Reply:
In that case, it might be better to revert to previous behavior and manually close the drawer when we trigger the edit.